frontend/src/concepts/pipelines/content/modelCustomizationForm/modelCustomizationFormSchema/validationUtils.ts

```diff
@@ -98,15 +105,24 @@ export const pipelineParameterSchema = z.record(z.string(), z.any()).superRefine
   }
 });
 
+export const trainingNodeSchema = z
+  .number()
+  .refine((val) => val > 0, { message: 'Number must be greater than 0' });
+
+export const storageClassSchema = z
+  .string()
+  .trim()
+  .min(1, { message: 'storage class is required' });
```

Comment:
```suggestion
  .min(1, { message: 'Storage class is required' });
```
All of our error messages seem to be capitalized. Should we follow this existing convention?